docs/multinode/config_rpm.md

```diff
@@ -0,0 +1,31 @@
+# RHEL with MicroShift RPMs
+Log into the `hypervisor host` and follow the instructions described in the
+[Getting Started with MicroShift](../getting_started.md) document to create the
+`microshift-pri` and `microshift-sec` virtual machines for running primary and
+secondary instances.
+
+> Until the MicroShift 4.14 software is released, it is necessary to compile the
+> MicroShift RPMs from the latest sources on the `development host` and install
+> them on the `microshift-starter` host.
+>
+> See the [RPM Packages](../devenv_setup.md#rpm-packages) documentation
```

Comment:
I think this part of the documentation needs some fleshing out. After following the instructions up to this point, I had two working microshift systems running an older build of microshift. First, I wiped away the old microshift via `echo 1 | sudo /usr/bin/microshift-cleanup-data --all"`. Next I built the microshift rpms from this branch on a fresh microshift-dev VM, copied them over to both microshift-pri and microshift-sec, and installed them (`rpm -Uvh *.rpm` - also had to `dnf install greenboot` to satisfy that dependency), and then rebooted both. Afterward, I re-enabled microshift and refreshed the kubeconfig as described in getting_started.md. The end result of all this I now I have the two VMs both running the new microshift build.

Reply:
I agree. I will add more information here

Reply:
I updated the instruction - please see if this is better now.
Note that it's not necessary to clean the data before upgrading MicroShift.
Also, if you use `dnf localinstall` command instead of the `rpm`, the dependencies are pulled automatically.